core/frontend/src/tools/MeasureTool.ts

```diff
@@ -152,7 +152,7 @@ function adjustPoint(ev: BeButtonEvent, segments?: Array<Segment>, locations?: A
   return ev.point;
 }
 
-/** @alpha */
+/** @public */
```

Comment:
Can you add some documentation to these tools, like a description of what they do?

Reply:
done